lib/puppet/type/chronos_job.rb

```diff
@@ -23,6 +23,34 @@
     end
   end
 
+  newproperty(:arguments) do
+    desc "The command to execute in the job."
+    validate do |val|
+      unless val.is_a? Array
```

Comment:
Can we stick to munge and to_a for ONLY things that we want to be arrays.  Ruby won't error or create double nested arrays if you to_a an array and is allows people to send a string if their intention is to send only a single argument.  Seems more natural to me then having to remember to define single value arrays all the time.